bases/br_ms_sinasc/dicionario/table_config.yaml

```diff
@@ -1,85 +1,94 @@
-source_bucket_name: basedosdados-dev
-project_id_staging: basedosdados-dev
-project_id_prod: basedosdados-dev
-table_id: dicionario # AUTO GENERATED
-dataset_id: br_ms_sinasc # AUTO GENERATED
 
-url_ckan: https://basedosdados.org/dataset/br-ms-sinasc # AUTO GENERATED
-url_github: https://github.com/basedosdados/mais/tree/master/bases/br_ms_sinasc # AUTO GENERATED
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: br_ms_sinasc
 
-version: v1.0 # REQUIRED
-
-last_updated: 2021-09-12 # AUTO GENERATED
+table_id: dicionario
 
 # Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
 # Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
 # como usar os dados.
-# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
-description: |  # REQUIRED
-    Dicionário para a base do Sistema de Informações de Nascidos Vivos (SINASC)
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description: Dicionário da base de Nascidos Vivos - SINASC
+
+# A máxima unidade espacial que a tabela cobre.
+# Exemplo:
+#   - continent: south_america
+#     country: br
+spatial_coverage:
+    - continent: south_america
+      country: br
+
+# Anos cobertos pela tabela.
+# Preencher como lista de intervalos.
+# Exemplos: ['1995(1)2019'], ['2002(2)2010', '2016', '2020'].
+temporal_coverage:
 
-# Quem está completando esse arquivo config?
+# A unidade temporal com qual a tabela é atualizada.
+# Opções em 'https://basedosdados.org/api/3/action/bd_available_options'
+update_frequency: unique
+
+# Nível de observação da tabela: o que representa cada linha.
+observation_level:
+
+last_updated:
+    metadata: '2022-04-30'
+    data: '2022-04-30'
+    release:
+
+# Versão da tabela. Seguindo o padrão de semantic versioning.
+# Exemplo: v1.1.3
+version: v1.0
```

Comment:
É uma correção apenas nos metadados? Porque se for deveria ser v1.1
Caso seja uma mudança no conteúdo também, acredito que deva ficar v.2.1

Reply:
No caso não houveram mudanças no dicionário, apenas atualização dos dados e table_config de microdados